Providers/Scripts/2.4x-2.5x/Scripts/nxOMSGenerateInventoryMof.py

```diff
@@ -148,7 +154,22 @@ def GenerateInventoyConfContents(FeatureName, Instances, RunIntervalInSeconds, T
   # Force upload even if the data has not changed \n \
   force_send_run_interval 24h \n \
   log_level warn \n \
-</filter> \n'
+</filter> \n \
+<match ' + Tag + '> \n \
+  type out_oms_' + Tag + '\n \
+  log_level trace \n \
+  num_threads 5 \n \
+  buffer_chunk_limit 5m \n \
+  buffer_type file \n \
+  buffer_path /var/opt/microsoft/omsagent/state/out_oms_' + Tag + '*.buffer \n \
```

Comment:
IMO - For the multihoming case, the buffer path should handle the <ws> in the path. As you do not want one workspace's data sitting in other workspace's buffer.

@lagalbra could you let me know how is the buffer path managed today?